packages/kit/src/views/Swap/pages/components/SwapActionsState.tsx

```
@@ -25,7 +27,7 @@ import {
 import { useSettingsAtom } from '@onekeyhq/kit-bg/src/states/jotai/atoms';
 import { ETranslations } from '@onekeyhq/shared/src/locale';
 import platformEnv from '@onekeyhq/shared/src/platformEnv';
-import { EModalRoutes, EOnboardingPages } from '@onekeyhq/shared/src/routes';
+import { EModalRoutes, EOnboardingPages, EOnboardingPagesV2, EOnboardingV2Routes, ERootRoutes } from '@onekeyhq/shared/src/routes';
```

ESLint

[error] Replace `·EModalRoutes,·EOnboardingPages,·EOnboardingPagesV2,·EOnboardingV2Routes,·ERootRoutes·` with `⏎··EModalRoutes,⏎··EOnboardingPages,⏎··EOnboardingPagesV2,⏎··EOnboardingV2Routes,⏎··ERootRoutes,⏎`

(prettier/prettier)